browser/ethereum_remote_client/ethereum_remote_client_service.cc

```diff
@@ -13,7 +13,6 @@
 #include "base/files/file_util.h"
 #include "base/functional/bind.h"
 #include "base/strings/string_util.h"
-#include "base/strings/stringprintf.h"
 #include "base/task/thread_pool.h"
```

Comment:
This will conflict with https://github.com/brave/brave-core/pull/29135/files
fyi

Reply:
Noted